### ISSUE TITLE
Project doesn't compile with "platform = espressif32@>=6.1.0

### ISSUE THREAD

**Issue by casimirextreme** [tbnobody/OpenDTU#862]
### What happened?

The project doesn't compile today due to 

`platform = espressif32@>=6.1.0` on platformio.ini

It looks like the latest tag is now 6.2.0 (pushed 4h ago) and add a breaking change.

Forcing to use `platform = espressif32@=6.1.0` let the latest Opendtu master to compile properly

### To Reproduce Bug

Since espressif32 tag v6.2.0

### Expected Behavior

Have the code to compile properly

### Install Method

Self-Compiled

### What git-hash/version of OpenDTU?

90a0cb60c622c66f84d91bc102a5a7b424b26059

### Relevant log/trace output

```Shell
...
Archiving .pio/build/generic/lib930/libAsyncTCP.a
Compiling .pio/build/generic/lib506/ESP Async WebServer/AsyncWebSocket.cpp.o
Compiling .pio/build/generic/lib506/ESP Async WebServer/SPIFFSEditor.cpp.o
Compiling .pio/build/generic/lib506/ESP Async WebServer/WebAuthentication.cpp.o
Compiling .pio/build/generic/lib506/ESP Async WebServer/WebHandlers.cpp.o
Compiling .pio/build/generic/lib506/ESP Async WebServer/WebRequest.cpp.o
Compiling .pio/build/generic/lib506/ESP Async WebServer/WebResponses.cpp.o
Compiling .pio/build/generic/lib506/ESP Async WebServer/WebServer.cpp.o
Compiling .pio/build/generic/lib919/WiFiClientSecure/WiFiClientSecure.cpp.o
Archiving .pio/build/generic/lib789/libWiFi.a
Compiling .pio/build/generic/lib919/WiFiClientSecure/esp_crt_bundle.c.o
Compiling .pio/build/generic/lib919/WiFiClientSecure/ssl_client.cpp.o
In file included from /home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/Arduino.h:176,
                 from .pio/libdeps/generic/ESP Async WebServer/src/ESPAsyncWebServer.h:24,
                 from .pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:21:
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp: In member function 'const __FlashStringHelper* AsyncWebServerRequest::methodToString() const':
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:965:34: note: in expansion of macro 'F'
   if(_method == HTTP_ANY) return F("ANY");
                                  ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:966:38: note: in expansion of macro 'F'
   else if(_method & HTTP_GET) return F("GET");
                                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:967:39: note: in expansion of macro 'F'
   else if(_method & HTTP_POST) return F("POST");
                                       ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:968:41: note: in expansion of macro 'F'
   else if(_method & HTTP_DELETE) return F("DELETE");
                                         ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:969:38: note: in expansion of macro 'F'
   else if(_method & HTTP_PUT) return F("PUT");
                                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:970:40: note: in expansion of macro 'F'
   else if(_method & HTTP_PATCH) return F("PATCH");
                                        ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:971:39: note: in expansion of macro 'F'
   else if(_method & HTTP_HEAD) return F("HEAD");
                                       ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:972:42: note: in expansion of macro 'F'
   else if(_method & HTTP_OPTIONS) return F("OPTIONS");
                                          ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:973:10: note: in expansion of macro 'F'
   return F("UNKNOWN");
          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp: In member function 'const __FlashStringHelper* AsyncWebServerRequest::requestedConnTypeToString() const':
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:978:31: note: in expansion of macro 'F'
     case RCT_NOT_USED: return F("RCT_NOT_USED");
                               ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:979:31: note: in expansion of macro 'F'
     case RCT_DEFAULT:  return F("RCT_DEFAULT");
                               ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:980:31: note: in expansion of macro 'F'
     case RCT_HTTP:     return F("RCT_HTTP");
                               ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:981:31: note: in expansion of macro 'F'
     case RCT_WS:       return F("RCT_WS");
                               ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:982:31: note: in expansion of macro 'F'
     case RCT_EVENT:    return F("RCT_EVENT");
                               ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebRequest.cpp:983:31: note: in expansion of macro 'F'
     default:           return F("ERROR");
                               ^
Compiling .pio/build/generic/lib930/espMqttClient/MqttClient.cpp.o
Compiling .pio/build/generic/lib930/espMqttClient/Packets/Packet.cpp.o
Compiling .pio/build/generic/lib930/espMqttClient/Packets/Parser.cpp.o
Compiling .pio/build/generic/lib930/espMqttClient/Packets/RemainingLength.cpp.o
*** [.pio/build/generic/lib506/ESP Async WebServer/WebRequest.cpp.o] Error 1
Compiling .pio/build/generic/lib930/espMqttClient/Packets/String.cpp.o
In file included from /home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/Arduino.h:176,
                 from .pio/libdeps/generic/ESP Async WebServer/src/ESPAsyncWebServer.h:24,
                 from .pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:21:
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp: In static member function 'static const __FlashStringHelper* AsyncWebServerResponse::responseCodeToString(int)':
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:45:22: note: in expansion of macro 'F'
     case 100: return F("Continue");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:46:22: note: in expansion of macro 'F'
     case 101: return F("Switching Protocols");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:47:22: note: in expansion of macro 'F'
     case 200: return F("OK");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:48:22: note: in expansion of macro 'F'
     case 201: return F("Created");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:49:22: note: in expansion of macro 'F'
     case 202: return F("Accepted");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:50:22: note: in expansion of macro 'F'
     case 203: return F("Non-Authoritative Information");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:51:22: note: in expansion of macro 'F'
     case 204: return F("No Content");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:52:22: note: in expansion of macro 'F'
     case 205: return F("Reset Content");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:53:22: note: in expansion of macro 'F'
     case 206: return F("Partial Content");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:54:22: note: in expansion of macro 'F'
     case 300: return F("Multiple Choices");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:55:22: note: in expansion of macro 'F'
     case 301: return F("Moved Permanently");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:56:22: note: in expansion of macro 'F'
     case 302: return F("Found");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:57:22: note: in expansion of macro 'F'
     case 303: return F("See Other");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:58:22: note: in expansion of macro 'F'
     case 304: return F("Not Modified");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:59:22: note: in expansion of macro 'F'
     case 305: return F("Use Proxy");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:60:22: note: in expansion of macro 'F'
     case 307: return F("Temporary Redirect");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:61:22: note: in expansion of macro 'F'
     case 400: return F("Bad Request");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:62:22: note: in expansion of macro 'F'
     case 401: return F("Unauthorized");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:63:22: note: in expansion of macro 'F'
     case 402: return F("Payment Required");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:64:22: note: in expansion of macro 'F'
     case 403: return F("Forbidden");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:65:22: note: in expansion of macro 'F'
     case 404: return F("Not Found");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:66:22: note: in expansion of macro 'F'
     case 405: return F("Method Not Allowed");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:67:22: note: in expansion of macro 'F'
     case 406: return F("Not Acceptable");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:68:22: note: in expansion of macro 'F'
     case 407: return F("Proxy Authentication Required");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:69:22: note: in expansion of macro 'F'
     case 408: return F("Request Time-out");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:70:22: note: in expansion of macro 'F'
     case 409: return F("Conflict");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:71:22: note: in expansion of macro 'F'
     case 410: return F("Gone");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:72:22: note: in expansion of macro 'F'
     case 411: return F("Length Required");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:73:22: note: in expansion of macro 'F'
     case 412: return F("Precondition Failed");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:74:22: note: in expansion of macro 'F'
     case 413: return F("Request Entity Too Large");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:75:22: note: in expansion of macro 'F'
     case 414: return F("Request-URI Too Large");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:76:22: note: in expansion of macro 'F'
     case 415: return F("Unsupported Media Type");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:77:22: note: in expansion of macro 'F'
     case 416: return F("Requested range not satisfiable");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:78:22: note: in expansion of macro 'F'
     case 417: return F("Expectation Failed");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:79:22: note: in expansion of macro 'F'
     case 500: return F("Internal Server Error");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:80:22: note: in expansion of macro 'F'
     case 501: return F("Not Implemented");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:81:22: note: in expansion of macro 'F'
     case 502: return F("Bad Gateway");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:82:22: note: in expansion of macro 'F'
     case 503: return F("Service Unavailable");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:83:22: note: in expansion of macro 'F'
     case 504: return F("Gateway Time-out");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:84:22: note: in expansion of macro 'F'
     case 505: return F("HTTP Version not supported");
                      ^
/home/opendtu/.platformio/packages/framework-arduinoespressif32/cores/esp32/WString.h:38:42: error: cannot convert 'const char*' to 'const __FlashStringHelper*' in return
 #define F(string_literal) (string_literal)
                                          ^
.pio/libdeps/generic/ESP Async WebServer/src/WebResponses.cpp:85:22: note: in expansion of macro 'F'
     default:  return F("");
                      ^
*** [.pio/build/generic/lib506/ESP Async WebServer/WebResponses.cpp.o] Error 1
=========================================== [FAILED] Took 30.99 seconds ===========================================

Environment    Status    Duration
-------------  --------  ------------
generic        FAILED    00:00:30.993
====================================== 1 failed, 0 succeeded in 00:00:30.993 ======================================
```

**Comment by b-a-9-0** [comment]
Same here.

**Comment by tbnobody** [comment]
either wait for a fix in the upstream repo: https://github.com/espressif/arduino-esp32/issues/8108
or just replace this line
https://github.com/tbnobody/OpenDTU/blob/90a0cb60c622c66f84d91bc102a5a7b424b26059/platformio.ini#LL18C20-L18C20

with
`platform = espressif32@6.1.0`

If it's not fixed until tomorrow I will pin it to 6.1.0 myself.

**Comment by tbnobody** [comment]
Added a workaround in 09f8a633d7ffe6e6bfcddd29009795ffc5551fc1

Should be fixed upstream soon.
